rfcs/text/0013-multiple-files.md

```diff
@@ -0,0 +1,220 @@
+# 0013: Multiple files in an event
+
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **2 (Draft)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+Many security events refer to more than one file at the same time or contain duplicated attributes for files.
+Examples of this are file copies or modifications.
+ECS has current support for multiple users in an event that was originally introduced in a previous RFC, this builds on the ideas introduced there, but for files.
+
+## Fields
+
+The new fields discussed in this RFC are the following:
+
+- `file.target.*`
+- `file.changes.*`
+
+Note that `file` would need to be made reusable with the following entries.
+
+```YAML
+# schemas/file.yml excerpt
+  reusable:
+    top_level: true
+    expected:
+      # Added for this RFC
+      - at: file
+        as: target
+      - at: file
+        as: changes
+```
+
+Currently `file` has 21 top-level fields and has a good number of fields that can be nested under it such as `code_signature`, `hash`, `pe`, and `x509`, so we may want to clarify exactly which fields we want to include (maybe just the top-level?).
```

Comment:
Yes, I think _not_ having all of `file.*` nestings carry along to `file.target.*` and `file.changes.*` is sensible for this case.

I'll go back and see if there are any changes in the tooling necessary to support.